IssueTracker/IssueTracker.FileSystem/Data/MetaData.cs

```diff
@@ -0,0 +1,67 @@
+﻿
+using IssueTracker.FileSystem.Data.IData;
+using Microsoft.Azure.Cosmos.Table;
+using Models.Request;
+using Models.Response;
+
+namespace IssueTracker.FileSystem.Data;
+
+public class MetaData : IMetaData
+{
+    private readonly CloudTable _metaDataTable;
+    public MetaData()
+    {
+        var storageAccount = CloudStorageAccount.Parse("AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;DefaultEndpointsProtocol=http;BlobEndpoint=http://127.0.0.1:10000/devstoreaccount1;QueueEndpoint=http://127.0.0.1:10001/devstoreaccount1;TableEndpoint=http://127.0.0.1:10002/devstoreaccount1;");
```

Comment:
move into appsettings.json

Reply:
I moved

Reply:
remove the appsettings.json from the FileSystem project